### PR TITLE
Updated Readme with installation instructions

### PR DESCRIPTION
**Pull request by iharshit009**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +67,17 @@ Once in the virtual environment, use the command `pipenv install PACKAGE_NAME1 P
...
+#### For Windows 
+
+[Download](https://www.seleniumhq.org/download/) the web driver according to version/type of your browser. Make sure that EXE file downloaded is inside the the project folder. In your development environment you can run the py file.
+
+#### For Linux
+
+Follow this command in your development Environment in your project folder
+```
+$ sudo apt-get install python-pip
+$ sudo pip install selenium
```
Comment (thealphadollar) [inline README.md:82]: I don't think these instructions are enough as I got an error that chromedriver could not be found. Can you please look into this?

**Review by thealphadollar — CHANGES_REQUESTED** [review]

**Comment by iharshit009** [comment]
Sir actually the webdriver .exe file should be inside the folder where our
.py file is present...

Them it would work...

**Comment by thealphadollar** [comment]
Please add all instructions, including this, to the readme. One should be
able to blindly follow the instructions and get the application running.

**Comment by iharshit009** [comment]
Okay sir

**Comment by iharshit009** [comment]
@thealphadollar sir Is it still incomplete or any part not working 
or it needs any updates??

**Review by thealphadollar — COMMENTED** [review]
Great work.

**Comment by thealphadollar** [comment]
@iharshit009 I apologize for the late response. The PR looks good to me.

**Pull request merged**